src/utils/updates.ts

```diff
@@ -122,17 +121,19 @@ export const watchPurses = async (
   for await (const purses of iterateNotifier(n)) {
     setPurses(purses);
 
-    for (const purse of purses as PursesJSONState[]) {
-      const { brand, displayInfo, brandPetname: petname } = purse;
-      const decimalPlaces = displayInfo && displayInfo.decimalPlaces;
-      const assetKind = displayInfo && displayInfo.assetKind;
-      const newInfo: BrandInfo = {
-        petname,
-        assetKind,
-        decimalPlaces,
-      };
-
-      mergeBrandToInfo([[brand, newInfo]]);
+    if (purses?.length) {
```

Comment:
Why would it be empty? 
```js
if (!purses?.length) {
  console.warn('no purses from notifier');
  continue;
}
```
Should be first line to not setPurses if there aren't any

Reply:
Done. We init the notifier kit to `null` https://github.com/Agoric/agoric-sdk/blob/master/packages/web-components/src/keplr-connection/watchWallet.js#L36 and it was getting caught trying to iterate on the first null value. I'm not sure why this wasn't an issue before, or how to avoid it other than checking.